src/sagemaker/base_predictor.py

```diff
@@ -75,6 +75,10 @@ def content_type(self) -> str:
     def accept(self) -> Tuple[str]:
         """The content type(s) that are expected from the inference server."""
 
+    def __str__(self) -> str:
+        """Overriding str(*) method to make more human-readable."""
+        return stringify_object(self)
```

Comment:
this looks to me like a good improvement, but please double check with SDK team if that's considered backward incompatible please. The alternative to be to write a different serialization function.

Reply:
Agree this would be a good improvement, I dont see any negative implications of this.